base/abstractarray.jl

```diff
@@ -124,7 +124,7 @@ julia> length(A)
 length(t::AbstractArray) = (@_inline_meta; prod(size(t)))
 _length(A::AbstractArray) = (@_inline_meta; prod(map(unsafe_length, indices(A)))) # circumvent missing size
 _length(A) = (@_inline_meta; length(A))
-endof(a::AbstractArray) = (@_inline_meta; length(a))
+endof(a::AbstractArray) = (@_inline_meta; last(linearindices(a)))
```

Comment:
any backporting concern that these don't have the `@inline_meta` on release-0.5 ?